engine/sound/src/sound.cpp

```diff
@@ -477,6 +492,26 @@ namespace dmSound
         return result;
     }
 
+    uint32_t GetMixRate()
+    {
+        return g_SoundSystem->m_MixRate;
+    }
+
+    uint32_t GetDefaultFrameCount(uint32_t rate)
+    {
+        if (rate == 48000)
+            return 1024;
+
+        if (rate == 44100)
+            return 768;
```

Comment:
As I understand it, the previous value what empirically chosed. It is larger than 44100/60=735.